### ISSUE TITLE
TOPIC: Choose files in Attachment does not appear to be in the correct place 

### ISSUE THREAD

**Issue by ssin1901** [citizenos/citizenos-fe#329]
**What is the problem?**
In Safari, when the User clicks Attachments, a Choose files button appears in what appears to be the wrong place (see screenshot)

<img width="1387" alt="Screenshot 2023-11-13 at 12 26 44" src="https://github.com/citizenos/citizenos-fe/assets/72747107/350788ba-a4e1-4a63-b555-4572b8a5e482">

This is different in Google Chrome, where no such button appears (see screenshot)

<img width="1383" alt="Screenshot 2023-11-13 at 12 29 00" src="https://github.com/citizenos/citizenos-fe/assets/72747107/43111aa7-7f4b-47f8-8fb9-b834dd1c29b9">

**Why is this a problem?**
Confusing UX on Safari

**Possible solution.**
Choose files button should not appear

**Comment by ilmartyrk** [comment]
@ssin1901 I could not replicate it on my local safari machine, please confirm if it is still an issue and on what Safari version

**Comment by ssin1901** [comment]
@ilmartyrk issue occurs on Safari Version 17.3 on Mac OS Monterey 12.7.3. 

I could not replicate this issue on any other browser or instance of Safari on another device.

**Comment by ssin1901** [comment]
Confirmed, this is still an issue also in the test environment. Could replicate in Safari on Mac, iPhone and some older Android devices using Chrome (screenshot1, screenshot2, screenshot3). It works fine on modern Chrome mobile and Desktop.

![Check-add-attachment-in-mobile-1-mobile-webkit-win32](https://github.com/user-attachments/assets/41bea6f5-3464-4ad6-9289-26a3f4644e18)

![Check-add-attachment-in-mobile-1-iPhone-SE-win32](https://github.com/user-attachments/assets/3f603ec3-723b-45d6-b2ec-9d6ee1758039)

![Check-add-attachment-in-mobile-1-Galaxy-Note-3-win32](https://github.com/user-attachments/assets/48cc8348-76de-4b2a-ada4-58a77a38c3c7)

**Comment by BeccaMelhuish** [comment]
@ssin1901 Just to check - is the button functional? I.e. the user can do what they want to do but it looks odd/confusing, or it's broken?

**Comment by ssin1901** [comment]
@BeccaMelhuish thanks for checking! It doesn't function either (see screen recording)


https://github.com/user-attachments/assets/55a8f21e-d81e-499c-b063-8188ff826dc9

**Comment by ssin1901** [comment]
Test again @ssin1901

**Comment by ssin1901** [comment]
@ilmartyrk @BeccaMelhuish issue occurs intermittently. I tested again with Mac OS Safari and iOS and it worked fine with iOS but then did not work with Safari in Mac OS even in incognito mode.

I'm thinking that since it doesn't happen every time, maybe we should not investigate this before the launch?

https://github.com/user-attachments/assets/a003a216-28b1-4f67-bfe4-3bea48a592af

**Comment by ssin1901** [comment]
@BeccaMelhuish this is a bug in Safari, should we move this to "Soon"? Reoccurrence confirmed

**Comment by BeccaMelhuish** [comment]
@ssin1901 Agreed, as adding attachements is quite important and 1 in 5 people are using Safari, then 'soon' makes sense :)
